MC/bin/o2dpg_sim_workflow.py

```diff
@@ -643,6 +644,16 @@ def createRestDigiTask(name, det='ALLSMALLER'):
          workflow['stages'].append(t)
          return t
 
+      elif det=='TOF':
+         if usebkgcache:
+            tneeds += [ BKG_HITDOWNLOADER_TASKS[det]['name'] ]
+         t = createTask(name=name, needs=tneeds,
+                     tf=tf, cwd=timeframeworkdir, lab=["DIGI","SMALLDIGI"], cpu='1')
+         t['cmd'] = ('','ln -nfs ../bkg_Hits' + str(det) + '.root . ;')[doembedding]
+         t['cmd'] += '${O2_ROOT}/bin/o2-sim-digitizer-workflow --use-ccdb-tof ' + getDPL_global_options() + ' -n ' + str(args.ns) + simsoption + ' --onlyDet ' + str(det) + ' --interactionRate ' + str(INTRATE) + '  --incontext ' + str(CONTEXTFILE) + ' --disable-write-ini' + putConfigValuesNew(["MFTAlpideParam, ITSAlpideParam, ITSDigitizerParam"])
+         workflow['stages'].append(t)
+         return t
```

Comment:
I think we can do it with less changes but I'll take care of it.

Reply:
thanks

Reply:
just a clarification.
Anchoring to ccdb works also for timestamp=0 (default entries are in place for any timestamp) so it works also for nightlies.